Remotion/ObjectBinding/Web/Res/HTML/BocAutoCompleteReferenceValue.jquery.js

```diff
@@ -1463,7 +1463,10 @@
                 field.selectionEnd = end;
             }
         }
-        field.focus();
+
+        if (focusAfterSelection && document.activeElement === document.getElementById(options.dropDownButtonID)) {

```

Comment:
this should be easier, not need to rely on activeELement. We know when the user triggered the dropdownbutton. this mens we know that the drop down button is the active element and therefor may set the focus to the field. There should also not be any addiitonal delay etc during which the user could click elsewhere while we process the first click, so this should alsways be straight forward. that the theory atleast.